source/aerobulk/flux.py

```diff
@@ -1,17 +1,57 @@
+import warnings
+
 import aerobulk.aerobulk.mod_aerobulk_wrap_noskin as aeronoskin
 import aerobulk.aerobulk.mod_aerobulk_wrap_skin as aeroskin
 import numpy as np
 import xarray as xr
 
 VALID_ALGOS = ["coare3p0", "coare3p6", "ecmwf", "ncar", "andreas"]
 VALID_ALGOS_SKIN = ["coare3p0", "coare3p6", "ecmwf"]
+VALID_VALUE_RANGES = {
+    "sst": [270, 320],
+    "t_zt": [180, 330],
+    "hum_zt": [0, 0.08],
+    "u_zu": [-50, 50],
+    "v_zu": [-50, 50],
+    "slp": [80000, 110000],
+    "rad_sw": [0, 1500],
+    "rad_lw": [0, 750],
+}
 
 
 def _check_algo(algo, valids):
     if algo not in valids:
         raise ValueError(f"Algorithm {algo} not valid. Choose from {valids}.")
 
 
+def _check_value_range(*args):
+    """Checks the input ranges for input fields"""
+    # parse inputs to names
+    args_dict = {
+        k: v
+        for k, v in zip(
+            ["sst", "t_zt", "hum_zt", "u_zu", "v_zu", "slp", "rad_sw", "rad_lw"], args
+        )
+    }
+    for var, data in args_dict.items():
+        # check for misaligned nans
+        if var != "sst":
+            if np.isnan(data).any():
+                raise ValueError(
+                    f"Found nans in {var} that do not align with nans in `sst`.\nCheck that nans in all fields are matched."
```

Comment:
I'm wonder if this `\n` is causing a mismatch in the `pytest.raises()` function in the tests below, causing the CI tests to fail. If so, then this suggestion should do the trick.

```suggestion
                    f"Found nans in {var} that do not align with nans in `sst`. Check that nans in all fields are matched."
```